components/workflow-mgt/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/internal/WorkflowImplServiceComponent.java

```diff
@@ -160,15 +165,14 @@ private void addDefaultBPSProfile() {
                 bpsProfileDTO.setProfileName(WFConstant.DEFAULT_BPS_PROFILE);
                 if (currentBpsProfile == null) {
                     workflowImplService.addBPSProfile(bpsProfileDTO, MultitenantConstants.SUPER_TENANT_ID);
-                    if (log.isDebugEnabled()) {
-                        log.info("Default BPS profile added to the DB");
-                    }
-                } else if (!currentBpsProfile.getWorkerHostURL().equals(url)) {
+                    log.info("Default BPS profile added to the DB.");
+                } else {
                     workflowImplService.updateBPSProfile(bpsProfileDTO, MultitenantConstants.SUPER_TENANT_ID);
+                    log.info("Default BPS profile updated.");
```

Comment:
This only happens in server startup only when details of server such as host url of admin username has updated, and hence logged in info level.